src/Cdn/Cdn/ChangeLog.md

```diff
@@ -19,8 +19,6 @@
 -->
 
 ## Upcoming Release
-
-## Version 1.7.0
 * Added cmdlets to support new AFD Premium / Standard SKUs
```

Comment:
@dingmeng-xue / @VeryEarly to confirm this change satisfies the ChangeLog format